typescript/forms.d.ts

```diff
@@ -345,29 +345,52 @@ export type FormElementWithInput<DefaultValue> = {
 
 export type NumberElement = {
   type: 'number'
+  /** Prevent the user from entering a number less than this number. */
   minNumber?: number
+  /** Prevent the user from entering a number more than this number. */
   maxNumber?: number
+  /**
+   * If `true`, the input will display as a slider instead of a numeric input
+   * and the `sliderIncrement` is required.
+   */
   isSlider: boolean
   /** If true, the form will indicate the number is a currency value. */
   displayAsCurrency?: boolean
+  /**
+   * Required if `isSlider` is `true`, otherwise must be `undefined`. The
+   * increment the slider will go up and down by. Must be greater than 0 and
+   * supports decimals.
+   */
   sliderIncrement?: number
+  /**
+   * If true, the user will not be able to enter a decimal number, only integers
+   * are allowed.
+   */
```

Comment:
applies to slider also, should probably be specific so it gets the increment right